warp10/src/main/java/io/warp10/continuum/sensision/SensisionConstants.java

```diff
@@ -316,6 +316,11 @@
   public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS = "warp.standalone.delete.datapoints";
 
   /**
+   * Number of datapoints deleted by 'standalone' delete per owner and application
+   */
+  public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS_PEROWNER = "warp.standalone.delete.datapoints.perowner";
```

Comment:
I actually just need the owner, but added app for completeness. I'll make it a separate metric tomorrow :-)